markdown/2017-10-02-welcome.md

```diff
@@ -0,0 +1,380 @@
+- **Academic Events Coordinator**: Rachel Mekhtieva Lee
+- **Industrial Events Coordinator**: Yoon Kim
+- **Social Events Coordinator**: Diana Avram
+- **Events Officers**: Alessandro Bonardi, Jack Pordi, Rohan Pritchard
+- **Marketing Officer**: Inara Ramji
+- **Webmaster**: Elliot Greenwood
+- **Game Dev Officer**: Uno Perwironegoro
+
+Throughout the year you will also get to know your departmental representatives
+(Dep Reps). They represent the student body (you) to the department.
+
+This year they are:
+
+- **Computing**: Martin Zlocha
+- **Joint Maths and Computing (JMC)**: Rachel Mekhtieva Lee
+- **Electronics and Information Engineering (EIE)**: Mwanakombo Hussein
+
+For details on how to get involved, just get in touch with us. We're always
+looking for volunteers to help us set up and tear down the events. If you would
+like to volunteer, please join our [DoCSoc: Operations Facebook
+group](https://www.facebook.com/groups/1509850436007222/).
+
+## What Do We Do?
+DoCSoc is run by students, for students. With the generous support of our
+sponsors, we run events such as theatre trips, talks, and our infamous bar
+nights (psst, all drinks are on DoCSoc)! Our membership base is mostly comprised
+of Computing, JMC, and EIE students, but membership can also be purchased on the
+[union
+website](https://www.imperialcollegeunion.org/activities/a-to-z/computing) for
+anyone not on a DoC course. Why not get your friends on other courses to become
+members too?
+
+To stay up-to-date on the events that DoCSoc runs, keep an eye out for the
+weekly digest (you're reading it right now) which is baked and then delivered
+fresh to your inbox every week! Additionally, liking our [Facebook
+page](https://www.facebook.com/ICDocSoc/) will help you stay up to date on
+everything going on with DoCSoc.
+
+## Our Sponsors
+Without the support of our generous sponsors, much of what we do and what makes
+DoCSoc so special would not be possible! Sponsors host many of our talks and
+will occasionally let you know of their great job opportunities.
+
+Which brings me on to why DoCSoc gets sponsorship: It's important to realise
+that our sponsors are looking to hire YOU! You get to build your career while we
+get the money to run awesome social events. To kick things off, we will be
+hosting our annual Sponsors' Exhibition on the 12th October from 14.00-17.30 in
+the Great Hall (more details below).
+
+## Our Main Events
+
+### Bar Nights
+
+Oh, bar nights&hellip; Arguably one of DoCSoc's most known type of event, bar nights
+happen just about every month during term time. They're a great opportunity to
+talk to your coursemates and make new friends! Upon arrival, a volunteer will
+greet you and hand you tokens which can be redeemed for a vast array of drinks
+(both alcoholic and non-alcoholic) behind the bar. Ran out of tokens? Depending
+on the event, just ask a committee member to give you a few more tokens. We
+don't bite!
```

Comment:
Maybe add something like "they will be wearing DoCSoc t-shirts with their name on the back"

Reply:
Not necessarily though. Don't Ops people volunteer too sometimes or is it only committee that hand out tokens?